build.gradle

```diff
@@ -182,6 +182,7 @@ dependencies {
   compile group: 'net.javacrumbs.shedlock', name: 'shedlock-provider-jdbc', version: '1.0.0'
   compile group: 'com.microsoft.azure', name: 'azure-storage', version: '8.0.0'
   compile group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.8', withoutJsonSmart
+  compile group: 'com.microsoft.azure', name: 'azure-servicebus-spring-boot-starter', version: '2.0.6', withoutJsonSmart
```

Comment:
is it compatible with the one above? (`compile group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.8', withoutJsonSmart`)

Reply:
Yes, I believe it is and it's what we also use in orchestrator.

Reply:
strange. i would imagine _spring boot starter_ is the only prerequisite